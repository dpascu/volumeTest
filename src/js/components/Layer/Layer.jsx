import React from 'react';
import PropTypes from 'prop-types';

const Layer = ({ children }) => (
  <div className="layer">
    {children}
  </div>
);

Layer.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Layer;
