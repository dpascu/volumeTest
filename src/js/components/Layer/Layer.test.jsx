/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';

import Layer from './Layer';

describe('Component Layer', () => {
  it('should render component', () => {
    const component = renderer.create(
      <Layer>
        Layer children
      </Layer>
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
