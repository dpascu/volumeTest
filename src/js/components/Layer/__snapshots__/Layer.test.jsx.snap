// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Layer should render component 1`] = `
<div
  className="layer left-up"
>
  Layer children
</div>
`;
